### ISSUE TITLE
unable use in the Ubuntu

### ISSUE THREAD

**Issue by CornerOfSkyline** [austin-----/code-gnu-global#4]
I install the VSCode and C++ intellisence in the Ubuntu and install the GNU Global, but it's not work.

**Comment by austin-----** [comment]
Could you please elaborate how it is not working? Did you run gtags in your project directory already? Does global work in the command line?

**Comment by badaix** [comment]
I might have a similar problem on Linux Mint. I get the following error message:

```
[Plugin Host] Activating extension `austin.code-gnu-global` failed:  Cannot find module './features/completionItemProvider'
(anonymous function) @ workbench.main.js:67
emitTwo @ events.js:87
emit @ events.js:172
handleMessage @ internal/child_process.js:685
channel.onread @ internal/child_process.js:440
workbench.main.js:67 
[Plugin Host] Here is the error stack:  Error: Cannot find module './features/completionItemProvider'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Function.D.n._load (/home/adminuser/Develop/VSCode-linux-x64/resources/app/out/vs/workbench/node/pluginHostProcess.js:19:15851)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/adminuser/.vscode/extensions/austin.code-gnu-global-0.1.4/out/src/extension.js:5:32)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

**Comment by austin-----** [comment]
It looks like a case sensitivity issue on Linux. The package apparently has the is file with an upper case file name. Will roll out a fix soon.

**Comment by austin-----** [comment]
Should have been fixed in 0.1.5. Please let me know if it is still an issue.

**Comment by badaix** [comment]
Thanks! It's working now.
Maybe you should mention somewhere in the Readme that Debian and it's derivates (Ubuntu, Mint, ...) are shipping an [outdated](https://bugs.launchpad.net/ubuntu/+source/global/+bug/1275029) version of GNU global (v5.7), which is throwing the error "--encode-path unrecognized"
